backend/docker.go

```diff
@@ -29,6 +29,7 @@ var (
 		"CERT_PATH":       "directory where ca.pem, cert.pem, and key.pem are located (default \"\")",
 		"CMD":             "command (CMD) to run when creating containers (default \"/sbin/init\")",
 		"MEMORY":          "memory to allocate to each container (0 disables allocation, default \"4G\")",
+		"SHM":             "/dev/shm to allocate to each container (0 disables allocation, default \"64MB\")",
```

Comment:
`humanize.ParseBytes` parses MB as 1000 * 1000 bytes, not 1024 * 1024 bytes, so if you say `SHM=64MB` you'll get something other than the default (64 * 1024 * 1024). I think 1024-based makes sense for memory things, so maybe change this string to say `default \"64MiB\"` instead?